src/sentry/static/sentry/app/components/modals/addDashboardWidgetModal.tsx

```diff
@@ -102,7 +102,12 @@ class AddDashboardWidgetModal extends React.Component<Props, State> {
     const {api, closeModal, organization, onAddWidget} = this.props;
     this.setState({loading: true});
     try {
-      const widgetData: Widget = omit(this.state, ['errors', 'loading']);
+      const widgetData: Widget = pick(this.state, [
```

Comment:
I was actually thinking of mentioning pick vs omit in the last PR but forgot to comment it

Reply:
Tony mentioned it, and then I forgot and merged the pull request, so that change ended up here.